src/pages/portfolio/ui/assets-table.tsx

```diff
@@ -0,0 +1,305 @@
+import { Table } from '@penumbra-zone/ui/Table';
+import { Card } from '@penumbra-zone/ui/Card';
+import { Text } from '@penumbra-zone/ui/Text';
+import { ArrowDownRight, ArrowUpRight } from 'lucide-react';
+import { Density } from '@penumbra-zone/ui/Density';
+import { useBalances } from '@/shared/api/balances';
+import { useAssets } from '@/shared/api/assets';
+import { observer } from 'mobx-react-lite';
+import { Skeleton } from '@/shared/ui/skeleton';
+import { ValueViewComponent } from '@penumbra-zone/ui/ValueView';
+import { connectionStore } from '@/shared/model/connection';
+import {
+  getMetadataFromBalancesResponse,
+  getBalanceView,
+} from '@penumbra-zone/getters/balances-response';
+import { BalancesResponse } from '@penumbra-zone/protobuf/penumbra/view/v1/view_pb';
+import { AssetIcon } from '@penumbra-zone/ui/AssetIcon';
+import { getDisplayDenomExponent } from '@penumbra-zone/getters/metadata';
+import { formatAmount } from '@penumbra-zone/types/amount';
+import { useRouter } from 'next/navigation';
+import { Button } from '@penumbra-zone/ui/Button';
+
+const LoadingState = () => {
+  return (
+    <div className='p-6'>
+      <Card>
```

Comment:
issue (ui / consistency): Card adds a default `gap-4`. You can either rely on it for margins, or add mt-* and md-* to inner elements, it's either of two. Right now, this section has large spacings because of gap being multiplied by inner margins.

On the screenshot, gaps are red and margins are orange

<img width="1187" alt="image" src="https://github.com/user-attachments/assets/61af7b84-5400-418d-8480-0a62292c1985" />

Reply:
Thanks, fixed this and made sure that the spacing is consistent between figma, loading state and normal state.